src/components/dialog/Dialog.jsx

```diff
@@ -15,7 +15,7 @@ export type DialogPropsType = $ReadOnly<{
   open: boolean,
   children: React.Node,
   size?: 's' | 'm' | 'l' | 'xl' | 'fullscreen',
-  reduceMotion?: boolean,
+  motionPreset?: 'none' | 'default',
```

Comment:
I think we still need the reduce motion mode, wdyt @coderitual?
It would be great to have ex.`motionPreset?: 'none' | 'reduce' | 'default'` to give more possibilities and be "accessibility-friendly"

Reply:
Yup for the future we may reintroduce `reduce` preset. For now though it's not required. Reduce motion should still work on css level though (so we respect user system setting). Exposing it on props level though was mainly for introducing user setting on application level but apparently reduce was mainly used as a replacement for "no motion".